src/modules/cmdpal/extensionsdk/BuildSDKHelper.ps1

```diff
@@ -1,102 +0,0 @@
-Param(
```

Comment:
@ericjohnson327 I suspect you'll end up bringing this back soon<sup>TM</sup>, but I think it's fine to prune this currently definitely unused version out. I'll let you commit a real version in the future ☺️